front/components/assistant/AssistantListActions.tsx

```diff
@@ -53,6 +55,7 @@ export default function AssistantListActions({
         title: `Assistant sharing updated.`,
         type: "success",
       });
+      onAssistantListUpdate?.();
```

Comment:
Not a big fan of this notation...

Reply:
You taught it to me :) I saw it in one of your PRs
No prob to change it to any other, I agree ; it's hard to find something concise and explicit at the same time

Reply:
😄. I got trick by the auto-save formatting. What about:
```suggestion
      onAssistantListUpdate && onAssistantListUpdate();
```